credentialz/credentialz.proto

```diff
@@ -224,6 +224,16 @@ service Credentialz {
   // Get the current public keys from the host. Each configured key will be
   // returned in the provided list.
   rpc GetPublicKeys(GetPublicKeysRequest) returns (GetPublicKeysResponse);
+
+  // Add a system role account.
```

Comment:
I'm not sure why my previous comment was marked as resolved, but I still would like to get the answer to that question.

Would be good to add a strict definition what a "system role account" is.

OpenConfig has SYSTEM_ROLE_ADMIN https://github.com/openconfig/public/blob/db78d44e93a97cef4db27f82bba3fba454e907a5/release/models/system/openconfig-aaa-types.yang#L77-L84, but I have a feeling that you mean something else in this context.

Reply:
>I think the intent of this is to provide a way to set a passwd/ssh-key/etc for:

is it? the proto says it can only add or delete account. for credentials management we have [gnsi.credz](https://github.com/openconfig/gnsi/tree/main/credentialz) service..